zerver/models.py

```diff
@@ -2253,12 +2253,17 @@ class Stream(models.Model):
 
     # Who in the organization has permission to send messages to this stream.
     stream_post_policy: int = models.PositiveSmallIntegerField(default=STREAM_POST_POLICY_EVERYONE)
-    STREAM_POST_POLICY_TYPES = [
-        STREAM_POST_POLICY_EVERYONE,
-        STREAM_POST_POLICY_ADMINS,
-        STREAM_POST_POLICY_RESTRICT_NEW_MEMBERS,
-        STREAM_POST_POLICY_MODERATORS,
-    ]
+    POST_POLICIES: Dict[int, str] = {
```

Comment:
I added a comment here and in the frontend about keeping these strings matching.